src/collection/collection.types.ts

```diff
@@ -10,7 +10,7 @@ import {
 export type Collection = (
 	| GetPublicCollectionsByIdQuery
 	| GetPublicCollectionsByTitleQuery
-)['app_collections'][0];
+)['app_collections_overview'][0];
```

Comment:
why do these queries use the overview table? Do we need to the share_type somewhere for these?

Reply:
Yes is needed, I got typing issues with Collection